actions/ctl.go

```diff
@@ -196,7 +196,7 @@ func parseCtl(data string) (ctlFunctionType, string, variables.RuleVariable, str
 			colname = spl3[0]
 			colkey = spl3[1]
 		} else {
-			colkey = spl3[0]
+			colname = spl3[0]
```

Comment:
This is the key change for fixing `941180-7`

Reply:
Can be another PR, but these lines can probably all be simplified to one

```
colname, colkey, _ := strings.Cut(spl2[1], ":")
```

so maybe a good time to clean it up

https://pkg.go.dev/strings#Cut

Reply:
I really think we need a unit test for this.